Gopkg.toml

```diff
@@ -28,3 +28,14 @@
   name = "github.com/aws/aws-lambda-go"
   version = "1.8.1"
 
+[[constraint]]
+  name = "github.com/DATA-DOG/go-sqlmock"
+  version = "1.3.3"
+
+[[constraint]]
+  name = "github.com/lib/pq"
+  version = "v1.0.0"
+
+[[constraint]]
+  name = "github.com/lib/errors"
```

Comment:
FYI, https://godoc.org/golang.org/x/exp/errors is incubating which _should_ inter-operate with Dave's pkg/errors. Given that, it might be time to start making use of this library more standard for our projects. We haven't used it historically in favor of sticking to standard library support. However, the combined features of pkg/errors and x/errors would make a lot of the kinds of error handling we do a bit easier and less tedious.